python/cuopt_server/cuopt_server/utils/data_definition.py

```diff
@@ -343,7 +345,12 @@ class LogResponseModel(StrictModel):
             "application/json": {
                 "examples": {
                     "Healthy response": {
-                        "value": {"status": "RUNNING", "version": "25.10"}
+                        "value": {
+                            "status": "RUNNING",
+                            "version": ".".join(
+                                cuopt.__version__.split(".")[:2]
```

Comment:
Since this is referenced in multiple places and should be cheap to implement, and since I imagine `cuopt-server` is more sensitive to startup time than `cuopt` itself, it might be worthwhile to move this to `cuopt`, like this:

```python
__version_major_minor__ = ".".join(__version__.split(".")[:2])
```

Then all of these places could just do

```python
from cuopt import __version_major_minor__
```

Reply:
Awesome suggestion, will add this.